servicenow/check.py

```diff
@@ -0,0 +1,199 @@
+# (C) Datadog, Inc. 2010-2016
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# stdlib
+
+# 3rd party
+import requests
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+from requests.packages.urllib3 import disable_warnings
+
+# Disable https warnings
+disable_warnings(InsecureRequestWarning)
+
+# project
+from checks import AgentCheck, CheckException
+
+EVENT_TYPE = SOURCE_TYPE_NAME = 'servicenow'
+
+
+class ServicenowCheck(AgentCheck):
+
+    INSTANCE_TYPE = "servicenow_cmdb"
+    SERVICE_CHECK_NAME = "servicenow.cmdb.topology_information"
+    service_check_needed = True
```

Comment:
The check configuration allows multiple servicenow instances to be defined, but the data associated with it is global to the check, meaning instances shared all these fields that are on the check now. We should either disallow multiple instances, or make sure that when multiple instances are defined, we properly isolate the data